functions/supplier/kafka-supplier/README.adoc

```diff
@@ -0,0 +1,28 @@
+# Apache Kafka Consumer Supplier
+
+A `Consumer<Message<?>>` that allows to publish messages to Apache Kafka topic.
```

Comment:
I believe this was copy/pasta from the `kafka-publisher` as it does not talk of supplier (but rather consumer).